scripts/install

```diff
@@ -184,6 +186,7 @@ ynh_systemd_action --service_name=$app --action=restart
 ynh_script_progression --message="Configuring permissions..."
 
 [ $is_public -eq 1 ] && ynh_permission_update --permission="main" --add="visitors"
+ynh_app_setting_set --app=$app --key=skipped_uris --value="/api/"
```

Comment:
Seem to be the old way and is deprecated. 
I suggest : `ynh_permission_create --permission="api" --label="api" --url="/api" --allowed="visitors" "all_users" --auth_header="false" --show_tile="false" --protected="true"`
What's you opinion ?

Reply:
I'll research it and give it a try.
Thanks for the review!